test/test_file_blob.rb

```diff
@@ -667,6 +667,8 @@ def test_language
           assert blob.language.nil?, "A language was found for #{filepath}"
         elsif language == 'Generated'
           assert blob.generated?, "#{filepath} is not a generated file"
+        elsif language == 'Generic'
+          assert !blob.language, "#{filepath} should not match a language"
```

Comment:
What's the difference between `Generic` and `Data`? Couldn't these files be moved to the `Data/` directory?

Reply:
There's no logical order to the contents of the `test/fixtures/*` subdirectories. In fact, our entire test-suite is a mess of badly-named-everything. Even worse, some of the helper methods **require** the current structure, so, could we just leave this mess as-is…?

Reply:
I'm not sure I understand to what lack of logical order you are referring. Could you give some examples? I was under the impression files were organized in `test/fixtures/[Language]` directories with exceptions for files of unknown/unclassified languages and binary files.

Reply:
`test/fixtures/Data` also contains obvious text-based formats that don't fit anywhere else.

I think we should add a `Generic` subdirectory, though, organised by the actual extensions being tested:

~~~
./Generic
./Generic/sol
./Generic/sol/ignored.sol
./Generic/sol/Gerber.sol
./Generic/sol/…
~~~

This would require the assertions to be written out manually, but at least it's obvious where future fixtures need to be added…

Reply:
Ah, yes. That would give some structure to this new directory and then it makes sense to keep it separate from `Data` :+1: